vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/BranchingPoint.java

```diff
@@ -0,0 +1,72 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IBranchingPoint;
+import edu.asu.diging.vspace.core.model.IChoice;
+
+@Entity
+public class BranchingPoint extends Slide implements IBranchingPoint {
+
+    @Id
+    @GeneratedValue(generator = "id-generator")
```

Comment:
this needs to be the same as the name of the genericgenerator defined below I believe